packages/slice-machine/src/modules/userContext/index.ts

```diff
@@ -85,7 +92,32 @@ export const userContextReducer: Reducer<
         hasSeenTutorialsTooTip: true,
       };
     }
+    case getType(refreshStateCreator): {
+      return {
+        ...state,
+        authStatus: getAuthStatus(action.payload.clientError),
+      };
+    }
     default:
       return state;
   }
 };
+
+const getAuthStatus = (
+  clientError: ErrorWithStatus | undefined
```

Comment:
would `clientError?: ErrorWithStatus` also work, or does it need to explicitly receive undefined?

Reply:
would work both ways. 

But the way I see it `clientError?: ErrorWithStatus` means the clientError argument is optional and does not necessarily have to be passed on. `clientError: ErrorWithStatus | undefined` means that the method takes an argument (which could be undefined)

Think the method `getAuthStatus` should always take a parameter because it doesn't make sense to call  `getAuthStatus()`

Reply:
okay :)